test/C/src/serialization/include/CMakeListsExtension.txt

```diff
@@ -1,3 +1,4 @@
 find_package(std_msgs REQUIRED)
 
-ament_target_dependencies(${LF_MAIN_TARGET} PUBLIC std_msgs)
+# Tradeoff of not using ament described here: https://docs.ros.org/en/foxy/How-To-Guides/Ament-CMake-Documentation.html?highlight=ament_target_dependencies
+target_link_libraries(${LF_MAIN_TARGET} PUBLIC std_msgs)
```

Comment:
But this is a test for our ROS integration, not our build system.

Abandoning Ament would be altering our migration procedure.
There are also existing migrated ROS nodes that use `ament_target_dependencies` in the order of a couple of dozen nodes.

Reply:
Thanks for pointing that out. I'll look for another solution.

Reply:
The "solution" that I chose was to enable a deprecated feature of CMake. Since there isn't really any case where it makes sense to use the syntax that ament uses, I think it's fine to live with the deprecation warnings for as long as ament continues to use that syntax, even if the process of transitioning takes years.